archeo/inc/patterns/image-with-description-and-right-aligned-headline.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Image with description and right-aligned headline
+ */
+return array(
+	'title'      => __( 'Image with description and right-aligned headline', 'archeo' ),
+	'categories' => array( 'pages' ),
+	'content'    => '<!-- wp:columns {"align":"full"} -->
+	<div class="wp-block-columns alignfull"><!-- wp:column {"width":"33.33%"} -->
+	<div class="wp-block-column" style="flex-basis:33.33%"><!-- wp:cover {"overlayColor":"background","minHeight":670,"contentPosition":"bottom center","isDark":false} -->
+	<div class="wp-block-cover is-light has-custom-content-position is-position-bottom-center" style="min-height:670px"><span aria-hidden="true" class="wp-block-cover__background has-background-background-color has-background-dim-100 has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:heading {"level":3,"fontSize":"large"} -->
+	<h3 class="has-large-font-size" id="palais-du-gouverneur-a-uxmal-detail-de-la-porte-principale-1">' . esc_html__( 'Palais Du Gouverneur, à Uxmal, detail de la Porte Principale', 'archeo' ). '</h3>
```

Comment:
I think there should be an uppercase text transform on this H3, as well as being right aligned:

Current | Design
---|---
<img width="508" alt="Screen Shot 2022-02-09 at 9 24 52 AM" src="https://user-images.githubusercontent.com/5375500/153220810-1b34dc70-5e8a-4320-8057-312a83a58b97.png"> | <img width="599" alt="Screen Shot 2022-02-09 at 9 25 08 AM" src="https://user-images.githubusercontent.com/5375500/153220830-a9aabf3c-19eb-4e1a-9d69-8aebd4249a40.png">

The spacing may need to be adjusted after those changes as well.